libs/@local/hash-graph-types/README.md

```diff
@@ -0,0 +1,3 @@
+# HASH Graph types
+
+Type definitions for the type system and entity validation based on schemas.
```

Comment:
I wonder if we can make this a bit more descriptive. @CiaranMn for thoughts.

Reply:
We should also update the package metadata to reflect the revised description here when added.

Reply:
I propose to do this in a follow-up so we can get this PR out of the way.